packages/clay-card/src/Body.tsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+import React from 'react';
+
+import classNames from 'classnames';
+
+import Context from './Context';
+
+export interface CardBodyProps extends React.HTMLAttributes<HTMLDivElement> {}
+
+export const Body: React.FunctionComponent<CardBodyProps> = ({
```

Comment:
Go ahead and pass `React.HTMLAttributes<HTMLDivElement>` straight into FunctionComponent. No need to extend into an empty interface above.

Reply:
Done!